convert.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+
+'use strict'
+
+const converter = require('./lib/converter.js');
+const formats = converter.formats;
+
+const command = async (specFilePath, targetFormat) => {
+
+    if (targetFormat === true ) {
```

Comment:
Why do you need this comparison here? I think you're just fine with the other one below.

Reply:
the reason is, commanderjs allow you to pass flags with no values and in that case will send true as flag's value. If a user forgets to pass a value to the flag will get an error similar to this "Invalid target format: true" and this will be exposing an internal implementation issue not something the user passed to the cli

Reply:
Makes sense! Thanks for explaining!